client/components/mma/shared/benefits/BenefitsConfiguration.ts

```diff
@@ -26,6 +26,8 @@ export interface ProductBenefit {
 	isUnavailable?: boolean;
 }
 
+export const supporterPlusSwitch = [newsApp, adFree];
```

Comment:
```suggestion
export const supporterPlusSwitchBenefits = [newsApp, adFree];
```

might make it more obvious what this const is?